modules/openapi-generator/src/main/resources/JavaSpring/oneof_interface.mustache

```diff
@@ -0,0 +1,6 @@
+{{>additionalModelTypeAnnotations}}{{>generatedAnnotation}}{{>typeInfoAnnotation}}{{>xmlAnnotation}}
+public interface {{classname}} {{#vendorExtensions.x-implements}}{{#-first}}extends {{{.}}}{{/-first}}{{^-first}}, {{{.}}}{{/-first}}{{/vendorExtensions.x-implements}} {
```

Comment:
The `x-implements` is something that I added to the `JavaClientCodegen` class, so you'd probably need to add it in the `SpringCodegen` as well.

Reply:
Its work also without x-implements, i think
 implcm.getVendorExtensions().putIfAbsent("x-implements", implcm.getVendorExtensions().get("implements"));
from OneOfImplementorAdditionalData make that job.